app/components/AddressBar/AddressBar.jsx

```diff
@@ -65,7 +65,7 @@ export default class AddressBar extends Component {
         const { addTab } = this.props;
 
         const addATab = tab => {
-            addTab({ url: `safe-browser://${tab}`, isActiveTab: true });
+            addTab( { url: `safe-browser://${tab}`, isActiveTab: true, windowId: this.props.windowId } );
```

Comment:
@manavbp when grabbing props, we tend to use destructuring at the top of the func (see how we grab `addTab` above). This keeps code cleaner and is marginally more optimised.